### DISCUSSION TITLE
LDAP Integration to specific groups

### DISCUSSION THREAD

**Discussion by mepramo** [netbox-community/netbox-docker#600] [Getting Help] [answered]
Hi Team,

I am new user of Netbox and this is my first post. I was able to integrate with LDAP authentication. The documentation was pretty easy to understand and straightforward.

I would like to perform some level of permission restrictions using LDAP. 
Ex: People in Datacenter LDAP group can only see datacenter related objects in Netbox.
-The remote user gets auto-created in a specific group which has the restrictions applied.

Whats the best way to achieve this ?

**Comment by mpibpc-mroose** [comment, marked as answer]
The way you want to go is to create permissions and assign those permissions to groups. Then those groups can then be "mapped" to LDAP groups.

**Reply by mepramo** [reply]
Yes.. My question was more on how I can map the groups in Netbox to the Ldap group

**Reply by mpibpc-mroose** [reply]
Then the [documentation of django-auth-ldap](https://django-auth-ldap.readthedocs.io/en/latest/permissions.html#group-mirroring) would be helpful for you.
